base/cvd/cuttlefish/host/commands/cvd/cli/parser/load_config.proto

```diff
@@ -84,7 +86,7 @@ message Vsock {
 message Graphics {
   repeated Display displays = 1;
   optional bool record_screen = 2;
-  optional string gpu_mode = 3;
+  optional string gpu_mode = 3 [deprecated = true];
```

Comment:
go/cf-config-flags-list mentions this should still be present, but limited to "auto" or "off" rather than the full set of options supported by our internal tools.